### ISSUE TITLE
How to test when WebKit also supports FACE?

### ISSUE THREAD

**Issue by jpzwarte** [calebdwilliams/element-internals-polyfill#100]
As in Real Soon Now: https://twitter.com/rniwa_dev/status/1611516973770477568

How do you test a polyfill when all modern browsers support the API? Fix the playwright WebKit version to an older one?

**Comment by calebdwilliams** [comment]
Yeah, that's the best I can tell. Fortunately, this now works in Safari Tech Preview … https://codepen.io/calebdwilliams/pen/xxJjNgr?editors=1010
